src/receivers/SocketModeReceiver.ts

```diff
@@ -175,3 +212,31 @@ export default class SocketModeReceiver implements Receiver {
     });
   }
 }
+
+function prepareCustomRoutes(customRoutes: CustomRoute[]): CustomRoute[] {
```

Comment:
It seems that you are repeating the same function here. How about having a single function useable for the receivers? We have `src/receivers/render-html-for-install-path.ts` and `src/receivers/verify-request.ts`. In the same way, we can have `prepare-custom-routes.ts` or something like that.

Reply:
I actually overlooked the fact that we had helpers split out into other files. Completely agree and will relocate it!